packages/fractal-page-object/src/page-object.ts

```diff
@@ -142,7 +140,7 @@ export default class PageObject extends ArrayStub {
    */
   [CLONE_WITH_INDEX](index: number): PageObject {
     let Class = this.constructor as PageObjectConstructor<PageObject>;
-    return new Class('', this, index, this.rootElement);
+    return new Class('', this, index);
```

Comment:
Why is `this.rootElement` gone now?

Reply:
oh I see, maybe redundant

Reply:
wouldn't it possibly be a breaking change to remove it though?

Reply:
I think this is a holdover from some previous implementation or something, because the field was never accessed by anything. I suppose technically this could be a breaking change because of the types, but I'm not too concerned about it because it was documented as "for internal use" before.